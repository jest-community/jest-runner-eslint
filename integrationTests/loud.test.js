const runJest = require('./runJest');

it('Outputs warnings as console messages', () => {
  return expect(runJest('loud')).resolves.toMatchSnapshot();
});
