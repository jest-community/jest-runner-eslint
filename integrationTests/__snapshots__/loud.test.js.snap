// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Outputs warnings as console messages 1`] = `
"PASS integrationTests/__fixtures__/loud/__eslint__/file.js
  ● Console
  console.warn
    /mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/loud/__eslint__/file.js
      3:1  warning  Unexpected console statement  no-console
    ✖ 1 problem (0 errors, 1 warning)
  ✓ ESLint
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
